compiler/rustc_borrowck/src/lib.rs

```diff
@@ -814,9 +820,8 @@ impl<'a, 'mir, 'tcx, R> rustc_mir_dataflow::ResultsVisitor<'mir, 'tcx, R>
 
             TerminatorKind::UnwindResume
             | TerminatorKind::Return
+            | TerminatorKind::TailCall { .. }
             | TerminatorKind::CoroutineDrop => {
-                // Returning from the function implicitly kills storage for all locals and statics.
-                // Often, the storage will already have been killed by an explicit
                 // StorageDead, but we don't always emit those (notably on unwind paths),
                 // so this "extra check" serves as a kind of backup.
                 let borrow_set = self.borrow_set.clone();
```

Comment:
Were these 2 lines deleted by accident? The comment doesn't seem to make sense otherwise.

Reply:
This looks accidental, yes. Can you make a PR reverting this?

Reply:
Sure, #127625

Reply:
thanks